main.tf

```diff
@@ -0,0 +1,186 @@
+data "aws_region" "current" {}
+
+locals {
+  architecture_layer_suffix_map = {
+    x86_64 = "",
+    arm64  = "-ARM"
+  }
+  runtime_base = regex("[a-z]+", var.runtime)
+  runtime_base_environment_variable_map = {
+    nodejs = {
+      DD_LAMBDA_HANDLER = var.handler
+    }
+    python = {
+      DD_LAMBDA_HANDLER = var.handler
+    }
+  }
+  runtime_base_handler_map = {
+    nodejs = "/opt/nodejs/node_modules/datadog-lambda-js/handler.handler"
```

Comment:
We might have an issue when the user is not installing the layer, but the package, where the DD handler is different here, maybe we can change that in a future PR.

Reply:
Interesting, is that a common approach to not use the layers and instead install the package directly? We can gauge interest based on user feedback

Reply:
Sometimes customers do it, definitely another PR, not now, until someone asks us to.